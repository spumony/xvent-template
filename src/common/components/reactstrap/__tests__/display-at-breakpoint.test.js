import React from 'react';
import { render } from '@testing-library/react';

import DisplayAtBreakpoint from '../display-at-breakpoint';

describe('display-at-breakpoint', () => {
  it('should render properly', () => {
    const { container } = render(
      <DisplayAtBreakpoint breakpoint="sm">
        <p>This text is visible only at sm breakpoint</p>
      </DisplayAtBreakpoint>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
