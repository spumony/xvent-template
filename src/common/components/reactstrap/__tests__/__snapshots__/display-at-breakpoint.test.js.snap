// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display-at-breakpoint should render properly 1`] = `
<div
  class="d-none d-sm-block"
>
  <p>
    This text is visible only at sm breakpoint
  </p>
</div>
`;
